README.md

```diff
@@ -1,2 +1,39 @@
+<img src="https://cloud.githubusercontent.com/assets/668093/12567089/0ac42774-c372-11e5-97eb-00baf0fccc37.jpg" alt="OpenMRS"/>
+
 # openmrs-ocl-client
-OCL client for OpenMRS
+
+> OCL client for OpenMRS is a web service that will allow OpenMRS users to manage their concepts on the web,
+> allow reusability of the existing concepts and allow one to create their concepts if they are not in exsitance.
+
+## Build
+### Requirements
+
+- Node and Npm [Here](https://nodejs.org/en/download/).
+
+### Installation Process
+```
+- Fork the repository https://github.com/openmrs/openmrs-ocl-client#fork-destination-box
+- Clone the forked repository to your computer
+    # git clone [the url of the repository]
+- Run 
+    $ npm install(if using npm)
+    $ yarn install (if using yarn)
+```
+### Runnig tests
```

Comment:
There is a typo on Running tests. Could you rectify it.

Reply:
On it